programmers/완주하지-못한-선수.js

```diff
@@ -0,0 +1,19 @@
+function solution(participant, completion) {
+    const obj = {};
+    
+    for (let same of participant) {
+        obj[same] = obj[same] ? obj[same] + 1 : 1;
+    };
+    
+    for (let i of completion) {
+        obj[i] = obj[i] - 1;
+    };
```

Comment:
```suggestion
function solution(participants, completions) {
    const obj = {};
    
    for (let participant of participants) {
        obj[participant] = obj[participant] ? obj[participant] + 1 : 1;
    };
    
    for (let completion of completions) {
        obj[completion] = obj[completion] - 1;
    };
```
주로 배열은 변수명을 복수로 하기 때문에  원소 변수명을 s없이 많이 사용하고 있는데 아마 프로그래머스 자체에서 배열을 단수로 주었기 때문에 2번 째  for문 이름 짓기에 고민이 많이 되셨을 것 같아요. 프로그래머스에서 주는 변수명을 사용하지 않고 들어가는 매개변수 순서만 지켜주신다면 얼마든지 다른 변수명을 사용하실 수 있답니다. 다음에 문제 푸실 때 참고하세요 :)

Reply:
매개변수를 바꿔도 괜찮았었네요?! 변수명을 짓기 어려울때 참고해야겠어요 오늘도 리뷰 감사합니다 :)